src/docs/testing/ui-performance/index.md

```diff
@@ -82,7 +81,7 @@ Flutter’s profile mode compiles and launches your application almost
 identically to release mode, but with just enough additional
 functionality to allow debugging performance problems.
 For example, profile mode provides tracing information to
```

Comment:
Replace all references to the Observatory with 
https://github.com/flutter/devtools
Caution that the tool is still early is still appropriate for the devtools app.
The devtools app provides similar cpu/gpu performance profiling to the observatory.
Generally the observatory should be something only vm authors will open.